build.sbt

```diff
@@ -30,7 +30,7 @@ val installAll =
   s"""apk update && apk add bash curl &&
      |apk add --update ruby ruby-bundler ruby-dev &&
      |rm /var/cache/apk/* &&
-     |gem install brakeman""".stripMargin.replaceAll(System.lineSeparator(), " ")
+     |gem install brakeman --no-rdoc --no-ri""".stripMargin.replaceAll(System.lineSeparator(), " ")
```

Comment:
I used --no-document, plz update your branch with master, as we added the brakeman version and did some fixes to correcly test the tool in circleCI